permissions/permissions.js

```diff
@@ -0,0 +1,13 @@
+const e = require('express');
+const actions = require('../constants/constants')
+const roles = require('../constants/constants')
```

Comment:
If you console-log these I think you'll find they're not working quite right, I think you're looking for:
```suggestion
const {actions, roles} = require('../constants/constants')
```